### PR TITLE
Update rails/ruby and remove frontend dependencies

### PR DESCRIPTION
**Pull request by hurrycane**
Updated dependencies, cleaned up Gemfile. This is the PR  related to #25

### CODE REVIEW

**Comment by hurrycane** [comment]
Integration blew up because the Ruby version is wrong.

@robertdolca can you please change the Travis jobs to Ruby 2.2 ?

**Comment by robertdolca** [comment]
The version is specified in .travis.yml. After you update that the build should succed.

File: Gemfile [hunk 1]
```diff
@@ -1,36 +1,14 @@
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 source 'https://rubygems.org'
-ruby '2.1.5'
-gem 'rails', '4.1.4'
+ruby '2.2.0'
```
Comment (robertdolca) [inline Gemfile]: Heroku uses the latest minor version (2.2.1). In order to make it work we need to use the same.

Comment (hurrycane) [inline Gemfile]: OK.

**Comment by hurrycane** [comment]
@robertdolca Updated to ruby 2.2.1

**Pull request merged**